### ISSUE TITLE
API Request with spaces in params values

### ISSUE THREAD

**Issue by ezgektor** [cgrates/cgrates#249]
Hello.
I noticed, that requests like this one lead to the situation, that CGRateS fails:
```
{"id": 1, "method": "ApierV1.GetTPRate", "params": [{"TPid":"DEFAULT", "RateId":"VOICE_0.15_0.05_60_1_0 "}]}
```

In this request there is a space in the end of "RateId" value.
After i execute it, CGRateS's status is fail and i should start it again.

Can you please check if it is a bug or I should check all the requests for such problems before executing them.

**Comment by danbogos** [comment]
Hi @ezgektor 

Can you please attach the .strace file? In debian you can find it in /var/run/cgrates/cgrates.strace.

Thanks,
DanB

**Comment by ezgektor** [comment]
Hello @danbogos 
Here it is:

**Comment by danbogos** [comment]
OK, this looks like a bug. Will check it, thanks!

DanB

**Comment by rif** [comment]
please test this with the latest package as well

**Comment by ezgektor** [comment]
hi @rif 
I've tested this case with actual version and i have the next result
When I execute requests like:
```
{"id": 1, "method": "ApierV1.GetTPRate", "params": [{"TPid":"DEFAULT", "RateId":"VOICE_0.15_0.05_60_1_0  "}]}
```
it cuts all spaces and returns rate "VOICE_0.15_0.05_60_1_0".
And when I execute requests like:
```
{"id": 1, "method": "ApierV1.GetAccount", "params": [{"Direction":"*out", "Tenant":"tenant.mobi", "Account":"34680000000   "}]}
```
It says that can't find account with such spaces at the end.

But the main problem was solved, CGRateS doesn't crashes anymore. So I think, I should close an issue. Thank you for your help.
